src/components/App/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from './App';
+
+describe('renders App', () => {
+  const component = shallow(<App />);
+
+  it('should have a page header', () => {
```

Comment:
There are probably a few other things we could test about the App itself - we could check for our App's specific divs' presence by id or class, or for specific tags we would expect to be present in the case of the App loading normally, etc.